guides/v2.2/cloud/project/project-conf-files_services-elastic.md

```diff
@@ -14,7 +14,7 @@ functional_areas:
 *   Elasticsearch performs quick and advanced searches on products in the product catalog
 *   Elasticsearch analyzers support multiple languages
 *   Supports stop words and synonyms
-*   Indexing does not impact customers until reindex is completed
+*   Indexing does not impact customers until the reindex operation is completed
 
 We support Elasticsearch versions 1.4, 1.7, 2.4, and 5.2. The default version is 1.7. We support Elasticsearch for all environments starting with {{site.data.var.ece}} 2.1 and later. Refer to [Elasticsearch information](http://devdocs.magento.com/guides/v2.1/config-guide/elasticsearch/es-overview.html) to learn more. For {{site.data.var.ece}} 2.0.X, you can use [Solr](http://devdocs.magento.com/guides/v2.0/cloud/project/project-conf-files_services-solr.html).
```

Comment:
You need to break the symbolic link to the 2.1 version because 2.2 supports ES 5.x. The link to [Elasticsearch information] need to point to the 2.2 version.